api/app/src/test/kotlin/packit/integration/controllers/UserControllerTest.kt

```diff
@@ -0,0 +1,56 @@
+package packit.integration.controllers
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.springframework.test.context.TestPropertySource
+import org.springframework.test.context.jdbc.Sql
+import packit.integration.IntegrationTest
+import packit.integration.WithAuthenticatedUser
+import packit.model.CreateBasicUser
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+@TestPropertySource(properties = ["auth.method=basic"])
+@Sql("/test-users.sql")
```

Comment:
i think yeah just does something simialr to the other repository methods... You can choose to run it before every test.. but here it would just run once before the class as we don't need it to run before every test